### DISCUSSION TITLE
how to fix import module error  or how to use?

### DISCUSSION THREAD

**Discussion by lozn00** [acheong08/ChatGPT#1141] [Q&A] [answered]
i try download the project to local  and execute ```python setup.py install``` after open the ```ChatGPT\src\revChatGPT\V1.py``` but 
```py
from OpenAIAuth import Authenticator
from OpenAIAuth import Error as AuthError

```
but module cannot to download or cannot match version

**Comment by acheong08** [comment, marked as answer]
`pip3 install --upgrade revChatGPT OpenAIAuth`

**Reply by acheong08** [reply]
Don't use setup.py directly

**Reply by lozn00** [reply]
thank you

**Comment by lozn00** [comment]
but 
 execute  ```ChatGPT\src\revChatGPT\V1.py```
```
  File "D:\dev\ChatGPT\src\revChatGPT\V1.py", line 25, in <module>
    from .utils import create_completer
ImportError: attempted relative import with no known parent package
```

**Comment by lozn00** [comment]
thank you
